casepro/pods/tests/utils.py

```diff
@@ -23,7 +25,7 @@ def perform_action(self, type_, params):
         '''
         Returns a successful action result with a message containing the type and params.
         '''
-        return (True, {'message': 'Type %s Params %r' % (type_, params)})
+        return (True, {'message': 'Type %s Params %s' % (type_, json.dumps(params))})
```

Comment:
The `repr` isn't consistent between Python versions so using a JSON dump instead